tests/unit/backend/wmg/data/test_ontology_labels.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from backend.wmg.data.ontology_labels import ontology_term_label, gene_term_label
+
+
+class OntologyLabelTests(unittest.TestCase):
+    def test__ontology_term_label(self):
+        # A conservative, high-level check that all ontologies have been loaded, without checking explicit counts
+        # This test is fragile, in that changes to our ontologies may break these these tests in the future, but that
+        # should happen infrequently
```

Comment:
I think it should only happen when we update to the cellxgene schema, we may want to add a note to that repo (or the reqs file) to make necessary updates here on updates

Reply:
yeah, I'll look for a spot to document.  It will just require a pip version update for cellxgene-schema, assuming the _set_ and _format_ of the files do not change. It's a little fragile, but since the cellxgene-schema is versioned, I'm okay with it.

Reply:
I've added my TODO list, but will not address immediately.